includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Advanced Settings Fields file.
+ *
+ * @package ActivityPub
+ */
+
+namespace ActivityPub\WP_Admin;
+
+/**
+ * Advanced Settings Fields class.
+ */
+class Advanced_Settings_Fields {
+
+	/**
+	 * Initialize.
+	 */
+	public static function init() {
+		add_action( 'load-settings_page_activitypub', array( self::class, 'register_advanced_fields' ) );
+	}
+
+	/**
+	 * Register settings.
+	 */
+	public static function register_advanced_fields() {
+		\add_settings_section(
```

Comment:
Not sure if we've discussed this yet, but I think it would be preferable to have this as a section in the regular settings tab. We have limited horizontal space in the navigation, but unlimited vertical space.